src/composed/message/builder.rs

```diff
@@ -114,6 +114,16 @@ pub trait Encryption: PartialEq {
     fn is_plaintext(&self) -> bool;
 }
 
+/// Subpacket configuration, per signing key
+#[derive(Debug)]
+enum SubpacketConfig {
+    Implicit,
```

Comment:
Maybe we call this `Default` and `UserDefined`?
Also a great place to document the subpackets that get added by default

Reply:
also `derive(Default)`  please